src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,11 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    public Ball getRandomBall() {
+        ColorSupplier colorSupplier = new ColorSupplier();
+        Random random = new Random();
+        return new Ball(colorSupplier.getRandomColor(), random.nextInt(100));
+    }
```

Comment:
The method `getRandomBall()` creates a new instance of `ColorSupplier` and `Random` each time it is called. This can be inefficient if the method is called multiple times. Consider moving the instantiation of `ColorSupplier` and `Random` to the class level as private final fields to avoid creating multiple instances for each method call.